### ISSUE TITLE
Call to undefined function MongoDB\is_last_pipeline_operator_out()

### ISSUE THREAD

**Issue by erkinduran** [mongodb/mongo-php-library#274]
mongodb/src/Collection.php can not see functions.php. so it can not find MongoDB\is_last_pipeline_operator_out() function. Can you help me? I can not write this library from the begining

**Comment by jmikola** [comment]
> mongodb/src/Collection.php can not see functions.php that is same file.

I don't understand this statement. Can you rephrase and/or elaborate?

If `functions.php` has been included (either manually or by default via Composer's `vendor/autoload.php`),  the `MongoDB\is_last_pipeline_operator_out()` function will exist. The current test suite would not pass if this were not the case.

For those not using Composer, we also discuss manually including `src/functions.php` in the [installation documentation](https://docs.mongodb.com/php-library/master/tutorial/install-php-library/).

**Comment by erkinduran** [comment]
i mean the collection.php give error undefined function MongoDB\is_last_pipeline_operator_out(). but it is in functions.php where in same file.

**Comment by moderndeveloperllc** [comment]
@erkinduran Are you using [composer](https://getcomposer.org/doc/01-basic-usage.md) for your dependency management, or you using `include` statements? Can you use [gist](https://gist.github.com) to paste an example of the file that is not working?

As this is an implementation question, and not a bug, you may also get more people seeing this if you post your question on [StackOverflow](http://stackoverflow.com/search?q=mongodb+library+php).

@jmikola A side question - is the docs.mongodb.com URL ready to replace the github.io URL for documentation?

**Comment by jmikola** [comment]
> i mean the collection.php give error undefined function MongoDB\is_last_pipeline_operator_out(). but it is in functions.php where in same file.

If you did not install the library with [Composer](https://getcomposer.org/) and are not including the `vendor/autoload.php` file, it is your responsibility to include `functions.php` and other class files on your own. The PHP interpreter is not able to access a function or class simply because it exists in another file.

> A side question - is the docs.mongodb.com URL ready to replace the github.io URL for documentation?

@moderndeveloperllc: It is. I see there are still a few places where the github.io URL is mentioned in the repository. I'll swap that over to https://docs.mongodb.com/php-library/ now. Thanks for catching that.

**Comment by jmikola** [comment]
@erkinduran: Is there any follow-up to my previous question?
